model/version_db.go

```diff
@@ -257,6 +257,18 @@ func VersionUpdateOne(query interface{}, update interface{}) error {
 	)
 }
 
+func UpdateVersionProject(versionID string, pp *ParserProject) error {
+	if versionID == "" {
+		return nil
```

Comment:
This should probably return an error, not nil, or else a caller passing an empty version string will silently fail.